### PR TITLE
USHIFT-2071: LVMS docs should include cases where no vgs exist on the system

### PR DESCRIPTION
**Pull request by copejon**

### CODE REVIEW

**Comment by pmtk** [comment]
Should OSDOCS be used to update customer facing documentation? Maybe this needs USHIFT ticket

File: docs/contributor/storage/default_csi_plugin.md [hunk 1]
```diff
@@ -40,7 +40,11 @@ The user provided lvmd config should be written to the same directory as the Mic
 If there is only one volume group on the system, LVMS uses it by
 default. If there are multiple volume groups, and no configuration
 file, LVMS looks for a volume group named `microshift`. If there is no
-volume group named `microshift`, LVMS is disabled.
+volume group named `microshift`, LVMS is disabled.  MicroShift also checks
+for the presence of volume groups on the system via `vgs`. If none exist, then
```
Comment (pmtk) [inline docs/contributor/storage/default_csi_plugin.md:44]: I feel like following sentence is misplaced - whole paragraph is about scanning volume groups.

> MicroShift also checks for the presence of volume groups on the system via `vgs`.

I'd move it to beginning of the paragraph and clearly separate 3 cases:
- no vg -> no topolvm
- 1 vg -> topolvm
- many vgs, one of them named microshift -> topolvm

**Comment by copejon** [comment]
@pmtk
> Should OSDOCS be used to update customer facing documentation? Maybe this needs USHIFT ticket

+1 I'll open a ushift ticket for this and link it here. This is just the repo side of the what will also be added to the osdocs.

File: docs/contributor/storage/default_csi_plugin.md (hunk 1 above)
Comment (copejon) [reply docs/contributor/storage/default_csi_plugin.md]: +1

**Pull request merged**